tests/spread/extensions/gnome-3-28/task.yaml

```diff
@@ -28,18 +32,32 @@ restore: |
   . "$TOOLS_DIR/snapcraft-yaml.sh"
   restore_yaml "snap/snapcraft.yaml"
 
+  # Remove content snaps installed by test.
```

Comment:
This is not needed, it is done in the general restore (`restore-each` in `spread.yaml`)

Reply:
Thanks!